docs-translations/zh-CN/tutorial/online-offline-events.md

```diff
@@ -3,15 +3,15 @@
 
 *main.js*
 ```javascript
-const electron = require('electron');
-const app = electron.app;
-const BrowserWindow = electron.BrowserWindow;
+const electron = require('electron')
+const app = electron.app
+const BrowserWindow = electron.BrowserWindow
 
-var onlineStatusWindow;
-app.on('ready', function() {
-  onlineStatusWindow = new BrowserWindow({ width: 0, height: 0, show: false });
-  onlineStatusWindow.loadURL('file://' + __dirname + '/online-status.html');
-});
+var onlineStatusWindow
+app.on('ready', function () {
```

Comment:
In our more up to date docs we are using arrow functions instead of old style functions like this.  Is this something we want to fix in the translations or something we probably don't care about hat much 👍